src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs

```diff
@@ -47,35 +47,36 @@ public static AadIssuerValidator GetIssuerValidator(string aadAuthority)
             if (string.IsNullOrEmpty(aadAuthority))
                 throw new ArgumentNullException(nameof(aadAuthority));
 
-            if (s_issuerValidators.TryGetValue(aadAuthority, out AadIssuerValidator aadIssuerValidator))
+            string authority;
+            try
```

Comment:
Instead of the try/catch to create a Uri, could we use Uri.TryParse() ?

Reply:
Hm, yes, I'll try.